plugin/trino-hudi/src/main/java/io/trino/plugin/hudi/HudiSessionProperties.java

```diff
@@ -200,4 +206,9 @@ public static boolean isIgnoreAbsentPartitions(ConnectorSession session)
     {
         return session.getProperty(IGNORE_ABSENT_PARTITIONS, Boolean.class);
     }
+
```

Comment:
`isIgnoreAbsentPartitions` method is unused now.

Reply:
removed